src/content/blog/2023_qrack_raid_0.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "How to build a ~40 qubit simulator for less than $3000 (and then simulate 54 qubits on it)!"
+author: Dan Strano
+day: 25
+month: 12
+year: 2023
+---
+
+At 32-bit floating point precision (with 64-bit complex numbers), 32 *giga*bytes of memory will fit 32 qubits worth of state vector simulation. For each additional qubit desired, you need to _double_ the memory (compounded per additional qubit). 40 qubits, depending on precision, therefore requires 8 or 16 *tera*bytes (_256 or 512 times_ as much RAM). Unfortunately, typical ram DIMMs might come in 16 gigabyte increments (give-or-take up to a couple of factors of 2), and maybe you’ll fit 8 of them in a server motherboard, for 128 gigabytes. Alternatively, NVIDIA A100 GPUs might carry 80 gigabytes apiece, you might fit about 64 gigabytes worth of quantum state vector simulation in a single GPU, and then it will take 128 of these (with high-speed interconnects) to reach 8 terabytes, at a cost of thousands or tens of thousands of dollars per hour to rent. What is the hobbyist quantum computing researcher to do?
```

Comment:
This post start is already very technical. Is there a way to make it a bit more gentle for non experts?

One suggestion would be to repeat the question of the title, with a bit more context. For example, something like: ```
There is a fast pace in the qubit count growth of experiments on quantum computers. At the same time, there is a growing literature of classical simulation (_add links_) of such experiments, often involving approximate methods and high performance computing (HPC) resources. One could thus be wondering: 

_What is the limit of what can be achieved on a personal computer/with moderate cloud resources?_

In this blog post we provide a possible recipe to this answer, showing how to build a ~40 qubit simulator for less than $3000. 
```

Still, even with this preamble, I feel like the current initial text starts a bit technical. Could you maybe provide a bit of context on how to do the resource estimation, without introducing immediately technical jargon? That could help a lot!